gsa/src/web/pages/cpes/listpage.js

```diff
@@ -47,50 +67,195 @@ export const ToolBarIcons = props => (
   <ManualIcon page="managing-secinfo" anchor="cpe" title={_('Help: CPEs')} />
 );
 
-const Page = ({filter, onFilterChanged, onInteraction, ...props}) => (
-  <React.Fragment>
-    <PageTitle title={_('CPEs')} />
-    <EntitiesPage
-      {...props}
-      createFilterType="info"
-      dashboard={() => (
-        <CpesDashboard
+const CpesPage = () => {
+  // Page methods and hooks
+  const [downloadRef, handleDownload] = useDownload();
+  const [, renewSessionTimeout] = useUserSessionTimeout();
+
+  // Powerfilter
+  const fallbackFilter = Filter.fromString('sort-reverse=modified');
```

Comment:
Could you do me a favor and move this filter outside of the component function. currently it is re-created on every render which isn't necessary at all.